src/open_inwoner/utils/forms.py

```diff
@@ -6,6 +6,44 @@
 from django.utils.translation import gettext_lazy as _
 
 
+class ErrorMessageMixin:
+    custom_error_messages = {
```

Comment:
The name is confusing, because these are *not* the custom error messages supplied by the caller. In your tests these are the `default_error_messages`.